cgsmiles/tests/test_molecule_resolve.py

```diff
@@ -150,7 +150,22 @@ def test_generate_edge(bonds_source, bonds_target, edge, btypes):
                         'O H C H C H H H O H',
                         [(0, 1), (0, 2), (2, 3), (2, 4),
                          (4, 5), (4, 6), (4, 7), (2, 8), (8, 9)]),
-
+                        # THF like test case with double edge and squash operator
+                        ("{[#A]1[#B]1}.{#A=[!]COC[!],#B=[!]CCCC[!]}",
```

Comment:
```suggestion
                        ("{[#A]=[#B]}.{#A=[!]COC[!],#B=[!]CCCC[!]}",
```
Interesting corollary, what does the bond order between fragments mean when you use the smash operator?